crates/bevy_winit/src/winit_windows.rs

```diff
@@ -58,43 +58,42 @@ impl WinitWindows {
         // AccessKit adapter is initialized.
         winit_window_attributes = winit_window_attributes.with_visible(false);
 
+        let select_monitor = &match window.mode {
```

Comment:
This feels clearer as `selected_monitor` to me.

Reply:
And since it's an option, I prefer `maybe_selected_monitor` for clarity when reading the code.

Reply:
> This feels clearer as `selected_monitor` to me.

I did actually call it selected_monitor earlier before I undid and redid some stuff :)